src/addon/cc-addon-postgresql-options.js

```diff
@@ -0,0 +1,83 @@
+import './cc-addon-option-form.js';
+import '../molecules/cc-error.js';
+import { css, html, LitElement } from 'lit-element';
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+import { ccAddonEncryptionAtRestOption } from '../templates/cc-addon-encryption-at-rest-option.js';
+
+const ENCRYPTION_DOCUMENTATION_URL = 'https://www.clever-cloud.com/doc/addons/postgresql/#encryption-at-rest';
+
+/**
+ * A component that displays the available options of a PostgreSQL add-on.
+ *
+ * * 🎨 default CSS display: `block`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/addon/cc-addon-postgresql-options.js)
+ *
+ * ## Type definitions
+ *
+ * ```js
+ * interface Options {
+ *   encryption: boolean,
+ * }
+ * ```
+ *
+ * @prop {Boolean} encryptionEnabled - Enables encryption at rest.
+ * @prop {Number} encryptionPrice - Price of the encryption option.
+ * @prop {String[]} options - List of options to enable for this selection.
+ *
+ * @event {CustomEvent<Options>} cc-addon-postgresql-options:submit - Fires when the form is submitted.
+ */
+export class CcAddonPostgresqlOptions extends LitElement {
+
+  static get properties () {
+    return {
+      options: { type: Array, attribute: 'options' },
+      encryptionEnabled: { type: Boolean, attribute: 'encryption-enabled' },
+      encryptionPrice: { type: Number, attribute: 'encryption-price' },
+    };
+  }
+
+  constructor () {
+    super();
+    this.options = [];
+    this.encryptionEnabled = false;
+  }
+
+  _onFormOptionsSubmit ({ detail }) {
+    dispatchCustomEvent(this, 'submit', detail);
+  }
+
+  _getFormOptions () {
+    return this.options.map((option) => {
+      switch (option) {
+        case 'encryption': return ccAddonEncryptionAtRestOption(ENCRYPTION_DOCUMENTATION_URL, this.encryptionPrice);
+        default: return null;
```

Comment:
Not a great fan of one liners like this. Is ESLint OK with that? If so, let's keep it and I'll update the ESLint config :p

Reply:
Yes, eslint is ok :p

Reply:
OK so let's keep it like that, we'll update ESLint later.